tools/generators/dh-domain-generator/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "cli": "nx",
+  "$id": "dh-domain-generator",
```

Comment:
suggestion: I think the name fits well with one minor adjustment - `dh-new-domain-generator` to be more indicative. What do you think?